cmd/nerdctl/fanotify_freebsd.go

```diff
@@ -0,0 +1,26 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func runFanotifyAction(cmd *cobra.Command, args []string) {
+	logrus.Warnf("fanotify is unavailable in freebsd platform")
+}
```

Comment:
If it's only available in linux, maybe just create a `fanotify_linux.go` instead of having 4 files. As an example:

https://github.com/containerd/nerdctl/blob/ac25d353a947e8ee7f46e156754acae6c0c8bf04/cmd/nerdctl/apparmor_load_linux.go